doc/source/report/dynamics.rst

```diff
@@ -3,60 +3,151 @@
+
+The x-axis is the time range, and the default time range is from 40,000 to 40,100. It is because the behavior dynamics show clear differences across the strains during this time range and because the behavior dynamics have more active actions than the other time period. Users can, of course, define the time range of their interest. But they should be aware of the lower and upper bounds of the time range before doing comparisons. For the first strain (strain_num = 0), the time range should be between 1 and 92,400. For the second one (strain_num = 1), it should be between 1 and 90,000. The time range should be between 1 and 88,800 for the last strain (strain_num = 2). 
+
+Of note is that, based on our understadning and explanations from the research team, 0 indicates the time when a mouse wakes up and starts its day. The unit of time is understood as one second. Thus, the default time range (from 40,000 to 40,100) represents around 11 hours after the start of the day. Based on this defnintion of the time range, users can change it and compare simulated behavior dynamics across the three strains.     
+
+We do not define the y-axis for this comparison function, because our interest is to visually understand the dynamics of the four states during the given time period. For better understanding of the behavior dynamics, we assign different colors for different states. In the plot, blue represents IS, skyblue represents eating, yellow represents drinking, and red represents other activities in AS.
+
+
+Result
+------
+
+The problem we are insteresting in here is whether the three strains of mice are indeed acting differently in a time series manner. The behaviors are compared using the three simulation mice, each for one strain. Therefore, as the first step, the optimal time interval is selected so as to have the most-real simulation mouse that behaves the most similarly to its strain, which is evaluated using the score system we created. As a result, the best time interval length is selected for each strain as well as the corresponding simulation mice behavior and the comparison score.
+
+::
+
+    find_best_interval(data_df,strain=0)
+    # (600, array([0, 0, 0, ..., 0, 0, 0]), 0.70509736459572225)
+    find_best_interval(data_df,strain=1)
+    # (600, array([0, 0, 0, ..., 0, 0, 0]), 0.91423472578532516)
+    find_best_interval(data_df,strain=2)
+    # (1800, array([0, 0, 0, ..., 0, 0, 0]), 0.83975976073161329)
+
+
+As the scipt suggested, the best time interval selected is 600 seconds for the first strain which generates the simultation mouse that has 71% similarity compared to the real mice in that strain, and the simulation behavior is quoted in the numpy array format. Similarly, it shows the 600s optimal time interval with 91% similarity for the second strain and the 1800s optimal time interval with 84% similarity for the third strain. 
```

Comment:
"As the script suggested, the best time interval selected is 600 seconds for the first strain. This generates a simulation mouse that is 71% similar to the real mice in that strain. Note that the simulation behavior is quoted in the numpy array format. Similarly, it shows the 600s optimal time interval with 91% similarity for the second strain and the 1800s optimal time interval with 84% similarity for the third strain."